rspec-mode.el

```diff
@@ -225,6 +225,13 @@ info, are considered errors."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-holder-dirs '("app" "lib")
```

Comment:
@pedz `source dirs` raises the same objection as `code dirs`. You don't like `applike` dirs?

I believe it's more important for this name to hint at the difference between these dirs and ther other source dirs, rather than emphasize that these are source dirs, too.